utils/usePopper.ts

```diff
@@ -26,15 +38,16 @@ export function usePopper(options) {
     eventsEnabled,
     positionFixed
   } = props;
-  const [state, setState] = useState(() => ({
+
+  const [state, setState] = useState<IUsePopperState>(() => ({
     data: undefined,
     placement: undefined
   }));
-  const popperInstance = useRef();
+  const popperInstanceRef = useRef<PopperJS | null>();
```

Comment:
I renamed this value so that it is clear that this is a ref, and not an instance of `PopperJS`.  Making the name more clear should help prevent us from looking up `PopperJS` properties on the ref:

```js
popperInstance.scheduleUpdate //  ❌doesn't exist

popperInstance.current.scheduleUpdate // ✅exists
```